frontend/app/routes/public/renew/$id/adult-child/review-adult-information.tsx

```diff
@@ -378,17 +379,32 @@ export default function RenewAdultChildReviewAdultInformation() {
         <fetcher.Form onSubmit={handleSubmit} method="post" className="mt-8 flex flex-row-reverse flex-wrap items-center justify-end gap-3">
           <input type="hidden" name="_csrf" value={csrfToken} />
           {hCaptchaEnabled && <HCaptcha size="invisible" sitekey={siteKey} ref={captchaRef} />}
-          <LoadingButton
-            id="confirm-button"
-            name="_action"
-            value={FormAction.Submit}
-            variant="green"
-            disabled={isSubmitting}
-            loading={isSubmitting && submitAction === FormAction.Submit}
-            data-gc-analytics-customclick="ESDC-EDSC:CDCP Renew Application Form-Adult:Submit - Review your information click"
-          >
-            {t('renew-adult-child:review-adult-information.submit-button')}
-          </LoadingButton>
+          {hasChildren ? (
+            <LoadingButton
+              variant="primary"
+              id="continue-button"
+              name="_action"
+              value={FormAction.Submit}
+              disabled={isSubmitting}
+              loading={isSubmitting && submitAction === FormAction.Submit}
+              endIcon={faChevronRight}
+              data-gc-analytics-customclick="ESDC-EDSC:CDCP Online Application Form-Adult_Child:Continue - Review adult information click"
+            >
+              {t('renew-adult-child:review-adult-information.continue-button')}
+            </LoadingButton>
+          ) : (
+            <LoadingButton
+              id="confirm-button"
+              name="_action"
+              value={FormAction.Submit}
+              variant="green"
+              disabled={isSubmitting}
+              loading={isSubmitting && submitAction === FormAction.Submit}
+              data-gc-analytics-customclick="ESDC-EDSC:CDCP Online Application Form-Adult_Child:Submit - Review child(ren) information click"
+            >
+              {t('renew-adult-child:review-child-information.submit-button')}
+            </LoadingButton>
+          )}
```

Comment:
The hCaptcha component should be placed in this no children block when submitting the application since it should be hit once per application submission.

Also, according to designers, this piece of text should show in this scenario as well:
![image](https://github.com/user-attachments/assets/5f9ba508-63d5-454a-8428-4f25aedd3772)